bin/ocamlformat.ml

```diff
@@ -24,8 +24,8 @@ let format ?output_file ~kind ~input_name ~source conf opts =
   else
     let f =
       match kind with
-      | `Impl -> Translation_unit.parse_and_format_impl
-      | `Intf -> Translation_unit.parse_and_format_intf
+      | `Impl -> Translation_unit.parse_and_format Use_file
```

Comment:
you can get rid of `Impl/Intf` and simplify this bit, but you need to erase the type argument from the GADT by defining something like:

```
type kind :
  | Kind of _ fragment
```

And having the cmdliner term return that. This can also be used in `parse_and_format` instead of the `_` wildcard.

Reply:
Good idea, done.